libs/login/WordPressLoginFlow/build.gradle

```diff
@@ -58,21 +65,21 @@ dependencies {
     }
 
     implementation 'com.github.bumptech.glide:glide:4.10.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.10.0'
+    kapt 'com.github.bumptech.glide:compiler:4.10.0'
 
     // Dagger
-    implementation 'com.google.dagger:dagger:2.22.1'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.22.1'
+    implementation "com.google.dagger:dagger:$daggerVersion"
```

Comment:
Where does `$daggerVersion` come from? Shouldn't we include this in the library?

Reply:
I think it pulls the version from the default `build.gradle`. Thinking about this, it might not work cause it doesn't solve the Woo using the same library. I'll add it to the buildscript of the login library